packages/task/package.json

```diff
@@ -23,16 +24,20 @@
     "url": "https://github.com/benjamn/wryware/issues"
   },
   "scripts": {
-    "clean": "../../node_modules/.bin/rimraf lib",
-    "tsc": "../../node_modules/.bin/tsc",
-    "rollup": "../../node_modules/.bin/rollup -c",
-    "build": "npm run clean && npm run tsc && npm run rollup",
-    "mocha": "../../scripts/test.sh lib/tests.js",
+    "build": "npm run clean:before && npm run tsc && npm run rollup && npm run clean:after",
+    "clean:before": "rimraf lib",
+    "tsc": "npm run tsc:es5 && npm run tsc:esm",
+    "tsc:es5": "tsc -p tsconfig.es5.json",
+    "tsc:esm": "tsc -p tsconfig.json",
+    "rollup": "rollup -c rollup.config.js",
+    "clean:after": "rimraf lib/es5",
     "prepare": "npm run build",
-    "test": "npm run build && npm run mocha"
+    "test:cjs": "../../scripts/test.sh lib/tests/bundle.cjs",
+    "test:esm": "../../scripts/test.sh lib/tests/bundle.js",
+    "test": "npm run test:esm && npm run test:cjs"
   },
   "dependencies": {
-    "@wry/context": "file:../context",
+    "@wry/context": "*",
```

Comment:
If I'm understanding correctly, interdependencies between monorepo packages are supposed to use this catch-all `*` version constraint, rather than a `file:...` path.